src/core/reaction_ensemble.cpp

```diff
@@ -1308,8 +1308,9 @@ int WangLandauReactionEnsemble::do_reaction(int reaction_steps) {
     double minimum_wang_landau_potential = find_minimum_non_negative_value(
         wang_landau_potential.data(), wang_landau_potential.size());
     for (int i = 0; i < wang_landau_potential.size(); i++) {
-      if (wang_landau_potential[i] >= 0) // check for whether we are in the valid
-                                         // range of the collective variable
+      if (wang_landau_potential[i] >=
+          0) // check for whether we are in the valid
+             // range of the collective variable
```

Comment:
Questionable indentation here too. Put the `0)` on the `if`'s line and make the two-line comment a three-line comment.